shared/constants/fs.js

```diff
@@ -182,6 +196,20 @@ const splitTlfIntoUsernames = (tlf: string): Array<string> =>
     .replace(/#/g, ',')
     .split(',')
 
+export const tlfToPreferredOrder = (tlf: string, me: string): string => {
```

Comment:
we have another function which basically does this. we should unify them

Reply:
Which function?

Reply:
https://github.com/keybase/client/blob/7e9ad67c0f86a82649f2e81586986892adcdf6fa/shared/util/kbfs.js#L5

Reply:
Hmm, that function returns a general user list. But I'll move this one to `utils/kbfs.js`.